src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -6,7 +6,8 @@
 .interactive-session {
 	max-width: 850px;
 	margin: auto;
-	position: relative; /* For chat dnd */
+	position: relative;
+	/* For chat dnd */
```

Comment:
There is a bunch of funny formatting changes in this file. Do you some CSS formatter that we don't have?